tools/ci/pr_preview.py

```diff
@@ -292,139 +276,146 @@ def get_pull_requests_with_open_ref(self):
 def is_open(pull_request):
     return not pull_request['closed_at']
 
+
```

Comment:
Nit: PEP8 does require two blank lines between top-level functions/classes, but this file only has one empty line everywhere, so either drop the new empty lines or fix the other places for consistency.